source/members-for-hire/index.html.md.erb

```diff
@@ -0,0 +1,18 @@
+---
+title: Members for hire
+breadcrumb: Members for hire
+created_by:
+  email: james@lety.co
+---
+
+A list of LRUG members who are available to hire on a contract basis. These individuals usually attend LRUG [meetings](/meetings/), or have attended a few of them, or maybe even just one.  Maybe they just signed up to the mailing-list, whatever, it's not an exclusive club or anything.
+
+<ul>
+  <% data.hire.each do |member| %>
+    <li class="vcard">
+      <a class="url" href="<%= member.url %>" title="<%= member.url %>">
```

Comment:
I believe this is dictated by the directory name, currently the directory is named members-for-hire and that is the url structure.